docs/crawler.js

```diff
@@ -0,0 +1,112 @@
+// This is the code example for the crawler configuration file for https://crawler.algolia.com/
+new Crawler({
```

Comment:
See if you can strip out the `¶` sign at the end of search results. If necessary, contact Algolia.